### PR TITLE
enhancement in tracks fragment 

### PR DESCRIPTION
**Pull request by pkarira**
giphy

### CODE REVIEW

**Comment by yadav-rahul** [comment]
@pkarira This PR contains a lot of extra code! Can you please remove it?

File: android/app/build.gradle [hunk 1]
```diff
@@ -87,6 +87,7 @@ dependencies {
     compile 'com.jakewharton.timber:timber:4.1.0'
     compile 'com.jakewharton:butterknife:8.4.0'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
+    compile 'jp.wasabeef:recyclerview-animators:2.2.5'
```
Comment (the-dagger) [inline android/app/build.gradle]: Why not use the native animations provided? 
Also I don't think that there is a need for animations anyways.

File: android/app/local.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
```
Comment (the-dagger) [inline android/app/local.properties]: Please don't commit any extra file.

**Comment by the-dagger** [comment]
I'm really not sure, why would we be needing such animations in the first place.
For now, I'd suggest to try and make the app's UI more mature by making it adhere to the Material Design guidelines.
We can add animations and interpolations later on.
Closing this for now.

**Pull request closed without merging**